snuba/admin/views.py

```diff
@@ -129,6 +129,24 @@ def configs() -> Response:
         )
 
 
+@application.route("/configs/<config_key>", methods=["DELETE"])
+def config(config_key: str) -> Response:
+    if request.method == "DELETE":
+        user = request.headers.get("X-Goog-Authenticated-User-Email")
+        state.delete_config(config_key, user=user)
+
+        notification_client.notify(
+            RuntimeConfigAction.ADDED,
+            {"option": config_key, "old": None, "new": None},
```

Comment:
It might be worth logging the old value, in case we need to restore it.

Reply:
Yeah, unfortunately we don't get it back from the config API so we'd need to make a get call first for it. I decided to omit that for now.

Reply:
Is it a problem to fetch the value? This isn't a high traffic endpoint.

Reply:
No it's not, will add it